src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -85,7 +85,11 @@
             "--no-server",
             "-H:+ExitAfterRelocatableImageWrite",
             "-H:+SharedLibrary",
-            "-H:+AddAllCharsets"
+            "-H:+AddAllCharsets",
+            "-H:+ReportExceptionStackTraces",
+            "-H:+AllowIncompleteClasspath",
```

Comment:
Why do we need AllowIncompleteClasspath for this?

Reply:
Removed now (as decided during slack conversation)